types/vue-the-mask/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for vue-the-mask 0.11
+// Project: https://github.com/vuejs-tips/vue-the-mask
+// Definitions by: Dominik Schmidt <https://github.com/domschmidt>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import { Component, DirectiveFunction, PluginObject } from 'vue';
+
+declare namespace VueTheMaskPlugin {
+    interface VueStatic {
+        (): void;
+    }
+}
+
+interface VueTheMaskPlugin extends PluginObject<void> {
```

Comment:
`void` doesn't really make sense for the type argument for `PluginObject` here because it's used to describe the type of an argument to a function. I'd recommend using `undefined` instead.